attrs/utils.go

```diff
@@ -0,0 +1,20 @@
+package attrs
+
+import "strings"
+
+// Return name for a data attribute
```

Comment:
Go convention is to start comments with the function name. It's helps `godoc` generate documentation. Can we change the comments to `// DataAttr returns the name for a data attribute.` please?